src/plugins/unified_histogram/public/index.ts

```diff
@@ -8,6 +8,9 @@
 
 import { UnifiedHistogramPublicPlugin } from './plugin';
 
+export type { BreakdownFieldSelectorProps } from './chart';
+export { UnifiedBreakdownFieldSelector } from './chart';
```

Comment:
I think the bundle size problem is coming from the fact that `chart/index.ts` is imported here which in turn imports `chart/chart.tsx` (in a non-lazy way). The easiest fix is probably to define the lazy component in a separate TS module that doesn't import `chart/chart.tsx` and only re-export that on the top level.

Reply:
Thanks for the tip, it saved me a lot of time.